generators/app/index.js

```diff
@@ -108,6 +108,24 @@ module.exports = JhipsterGenerator.extend({
             }.bind(this));
         },
 
+        checkNode: function () {
+            if (this.skipChecks || this.skipServer) return;
+            var done = this.async();
+            exec('node -v', function (err, stdout, stderr) {
+                if (err) {
+                    this.warning('Node is not found on your computer.');
+                } else {
+                    var nodeVersionMajor = stdout.split('.')[0].replace(/v/g, '');
+                    var nodeVersionMinor = stdout.split('.')[1];
+                    var nodeVersion = stdout.replace(/\n/g, '');
+                    if (nodeVersionMajor < 6 || (nodeVersionMajor >= 6 && nodeVersionMinor < 9)) {
```

Comment:
my node version is v7.5.0 is least version but `yo jhipster` waring `). You should use at least Node v6.9.0+v7.5.0`
**(nodeVersionMajor >= 6 && nodeVersionMinor < 9)** is not ok